js/src/ui/Form/AddressSelect/addressSelect.js

```diff
@@ -210,6 +214,19 @@ class AddressSelect extends Component {
     );
   }
 
+  renderLoader () {
+    if (!this.store.loading) {
+      return null;
+    }
+
+    return (
+      <Loading
+        className={ styles.loader }
+        size={ 0.5 }
```

Comment:
Do we need to override the defaults? How does this work for consistency across the full UI?